domains/group.is-an.app.js

```diff
@@ -0,0 +1,14 @@
+addSubDomain({
+  description: 'Hi there', // describe your project in this field
```

Comment:
```suggestion
  description: 'Hi there',
```

Remove all comments (everywhere), please